From d417ad122a04c1a6d191928170aa1a6368229678 Mon Sep 17 00:00:00 2001
From: bndlfm <firefliesandlightningbugs@gmail.com>
Date: Mon, 16 Dec 2024 21:32:19 -0600
Subject: [PATCH] fixing openxr.pc.in

---
 src/loader/openxr.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/loader/openxr.pc.in b/src/loader/openxr.pc.in
index 9772bf1..376f221 100644
--- a/src/loader/openxr.pc.in
+++ b/src/loader/openxr.pc.in
@@ -4,7 +4,7 @@
 
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=@CMAKE_INSTALL_PREFIX@
-libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
 includedir=${prefix}/include
 
 Name: @CMAKE_PROJECT_NAME@
